test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -18,23 +18,30 @@ public void isAnyNull() {
         assertFalse(Utils.isAnyNull(new Object(), new Object()));
         assertFalse(Utils.isAnyNull("test"));
         assertFalse(Utils.isAnyNull(""));
+        assertFalse(Utils.isAnyNull(123));
+        assertFalse(Utils.isAnyNull(false));
 
```

Comment:
Can I ask if all test methods always follow the naming convention of test<MethodName>_<StateUnderTest>_<ExpectedBehavior>?

Does each sub-state have to always exist in a separate test method or can they exist in a single test method like shown here? In this case, since there are multiple states we omit the <StateUnderTest> and <ExpectedBehavior> but do we need to have the test method name as test<MethodName>?